src/components/Home/ClusterDashboardResourcesV5.js

```diff
@@ -0,0 +1,94 @@
+import * as actionTypes from 'actions/actionTypes';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  selectLoadingFlagByAction,
+  selectResourcesV5,
+} from 'selectors/clusterSelectors';
+import { Dot } from 'styles';
+import RefreshableLabel from 'UI/RefreshableLabel';
+
+import ClusterDashboardLoadingPlaceholder from './ClusterDashboardLoadingPlaceholder';
+import ClusterDashboardNodes from './ClusterDashboardNodes';
+
+function ClusterDashboardResourcesV5({
+  cluster,
+  loadingClusters,
+  loadingNodePools,
+  loadingStatus,
+  resources,
+}) {
+  const { memory, cores, numberOfNodes } = resources;
+  const hasNodePools = cluster?.nodePools && cluster.nodePools.length !== 0;
+  const loading = loadingClusters || loadingStatus || loadingNodePools;
+
+  if (loading) return <ClusterDashboardLoadingPlaceholder isV5Cluster={true} />;
+
+  return (
+    <div data-testid='cluster-resources'>
+      {numberOfNodes !== 0 && hasNodePools && (
+        <RefreshableLabel value={numberOfNodes}>
+          <span>{cluster.nodePools.length} node pools, </span>
+        </RefreshableLabel>
+      )}
+      <RefreshableLabel value={numberOfNodes}>
+        <ClusterDashboardNodes
+          numberOfNodes={numberOfNodes}
+          createDate={cluster.create_date}
+        />
+      </RefreshableLabel>
+      {numberOfNodes !== 0 && hasNodePools && (
```

Comment:
Since we're doing these 2 checks together in both cases, I think it would be sensible to add the `numberOfNodes !==` check to `hasNodePools`. What do you think?

Reply:
👍